### ISSUE TITLE
přeskočit/přetočit v Pluginu Prima PLAY nejde

### ISSUE THREAD

**Issue by synog** [kodi-czsk/plugin.video.dmd-czech.prima#3]
Ahoj, mám 1 problém. Když chci přeskočit/přetočit v Pluginu Prima PLAY v čase, tak se kousne obraz a po chvilce i zvuk a dál to nejde. 

Da se to upravit?

P.S.: Chci poděkovat všem co spolupracovali na českých pluginu ale hlavně na Prima PLAY.

Dekuji﻿

**Comment by ToKroCZ** [comment]
Souhlasím, videa z prima play nelze pretacet, za několik vteřin se vypne.
